system_tests/resources/start-ec2-worker-vm-blueprint.yaml

```diff
@@ -60,12 +64,23 @@ node_templates:
       aws_config: *AWS_CONFIG
 
   server:
-    type: cloudify.aws.nodes.Instance
+    type: cloudify.aws.nodes.WindowsInstance
     properties:
       install_agent: false
       image_id: { get_input: image_id }
       instance_type: { get_input: instance_type }
       aws_config: *AWS_CONFIG
+      parameters:
+        user_data: |
+          <powershell>
+          &winrm quickconfig -q
```

Comment:
are all of those really necessary?
talk with NirB, im pretty sure he's got a test where only one of these lines is used